package/MDAnalysis/analysis/rms.py

```diff
@@ -138,70 +137,90 @@
 logger = logging.getLogger('MDAnalysis.analysis.rmsd')
 
 
-def rmsd(a, b, weights=None, center=False):
+def rmsd(a, b, weights=None, center=True, superposition=False):
```

Comment:
Can we turn centering off by default, too? I'm of the opinion that by default `rmsd` should do no transformations of the coordinates at all, but should just take them as-is.